dbt-cta/dbt_project.yml

```diff
@@ -5,7 +5,7 @@ config-version: 2
 profile: 'default'
 
 model-paths: ["{{ env_var('SYNC_NAME') }}/models"]
-packages-install-path: "{{ env_var('SYNC_NAME') }}/dbt-packages"
+packages-install-path: "/tmp/{{ env_var('SYNC_NAME') }}/dbt-packages"
```

Comment:
oh this was just me noodling around 😅 if were gonna try out the whole deps already in gcs this will have to be something different maybe even just `dbt-packages`?

Reply:
lol I've been inconsistent about this part of the discussion between this PR and the other one! Anyway, just using `"dbt-packages"` (no slash) seems to work! Updated the code in this PR accordingly 😎